### ISSUE TITLE
Discussion of potential future goals for Flipflop?

### ISSUE THREAD

**Issue by rolftimmermans** [voormedia/flipflop#19]
The current goal of Flipflop is pretty simple: have a series of on/off switches that can be easily configured, and stored in whatever way you want.

Are there any opinions on how to move forward? Are there use cases that could be interesting to address? I'm thinking about things like incremental rollout, enabling features per user/language/country, etc.

Feel free to comment here with ideas – once they are concrete a new issue can be opened to discuss them in more detail.

**Comment by jcoyne** [comment]
I would love to see a release in the short term.  I'm really happy with where it is right now. Thanks for all your hard work.

@tpendragon do you have any ideas?

**Comment by tpendragon** [comment]
Nope, I'm pretty happy. The library's nice and extendable, and most of the above could probably be implemented as more complicated strategies. Short term, a release, and maybe https://github.com/projecthydra-labs/hyrax/blob/master/app/strategies/hyrax/strategies/yaml_strategy.rb in core is all I could want for the moment.

**Comment by mjgiarlo** [comment]
Agree with @jcoyne and @tpendragon! Thanks for all your work on this, @rolftimmermans. Would love to see a new release with all the latest changes.

**Comment by rolftimmermans** [comment]
A new 2.3.0 release is now available (you can see [all changes here](https://github.com/voormedia/flipflop/blob/master/CHANGES.md)).

I'm keeping this issue open for now to discuss any potential future ideas.

**Comment by americos** [comment]
Great work @rolftimmermans, replacing Flip with Flipflop has been super easy
